scripts/translation2json.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+"""
+translation2js.py
+This script is used to create the {lang}.json files from the csv.
+
+Usage: ./env/bin/python translation2js.py [CSV_FILE [JSON_FOLDER]]
+default: CSV_FILE: src/locales/translations.csv, JSON_FOLDER: src/locales/
+"""
+
+import argparse
+import csv
+import json
+import gspread
+import sys
+import os
+
+from os.path import join
+from oauth2client.client import SignedJwtAssertionCredentials
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+
+def main(args, delimiter=',', quotechar='"'):
+    translations = {}
+    if args.gspreads is not None:
+        for key in args.gspreads:
+            csv_reader = _get_csv_reader_from_gspread(
+                key,
+                args.key_file,
+                delimiter=delimiter,
+                quotechar=quotechar
+            )
+            translations = _process_csv_file(csv_reader, translations)
+
+    if args.files is not None:
+        for filename in args.files:
+            with open(filename, 'r') as csv_file:
+                csv_reader = csv.DictReader(csv_file, delimiter=delimiter, quotechar=quotechar)
+                translations = _process_csv_file(csv_reader, translations)
+
+    _save_translations(translations, args.output_folder)
+
+
+def _get_csv_reader_from_gspread(key, key_file, delimiter=',', quotechar='"', **kwargs):
+    if 'DRIVE_USER' in os.environ.keys() and 'DRIVE_PWD' in os.environ.keys():
+        drive_user, drive_password = os.environ['DRIVE_USER'], os.environ['DRIVE_PWD']
+        gc = gspread.login()
```

Comment:
In our version of the script, I changed how we get the document from the drive. Since your document is publicly available, I make a get on the export as CSV URL (https://docs.google.com/spreadsheets/d/1F3R46w4PODfsbJq7jd79sapy3B7TXhQcYM7SEaccOA0/export?format=csv&gid=0) to get the document. That allowed me to:
- Use your document without authentication
- Simplify the script.

Are you OK to drop authentication to access this document?